README.md

```diff
@@ -844,7 +843,7 @@ This is very similar to Travis CI. With the same **build.py** script we have the
 
     environment:
         PYTHON: "C:\\Python27"
-        PYTHON_VERSION: "2.7.8"
+        PYTHON_VERSION: "3.78"
```

Comment:
Does not match the `PYTHON` variable one line above

Reply:
Big Thank you! 👏